pallets/schemas/src/rpc/Cargo.toml

```diff
@@ -30,8 +30,6 @@ sp-blockchain = { git = "https://github.com/paritytech/substrate", default-featu
 sp-std = {default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.18"}
 pallet-transaction-payment = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.18"}
 pallet-transaction-payment-rpc-runtime-api = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.18"}
-
-[dev-dependencies]
```

Comment:
why does this need to be included in schemas?

Reply:
@enddynayn  my next PR removes it, this was just a first

Reply:
You need it for setting the Governance Schema Max as that's a sudo command, and for running it in the UI I assumed you would need it in regular dependences here too. I can try it with it back in dev-dependencies (because we needed it for testing)

Reply:
how do you use it in testing?

Reply:
taking it out, not needed.